### PR TITLE
add najaxjs v1.0.4 with npm auto-update config

### LINKED ISSUE #6868: [Request] Add najaxjs
Library name: najaxjs
Git repository url: https://github.com/any-js/najaxjs
npm package url(optional): https://www.npmjs.com/package/najaxjs
License(s): MIT
Official homepage: http://any-js.github.io/

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6868 
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/any-js/najaxjs
 * Official website (optional, not the repository): https://any-js.github.io/
 * NPM package url (optional): https://www.npmjs.com/package/najaxjs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1
   - Count of watchers: 1
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 7
   - Downloads in the last month: 118

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #6868, cc @anyjser

### CODE REVIEW

**Comment by x09326** [comment]
@kennynaoh 
The latest version of this lib has been v1.0.4.
Would you update the latest version?
Sorry, I'm late to review it.

**Comment by kennynaoh** [comment]
@x09326 
um..., the latest version published on npm yesterday.
And I have updated it.
Please help me review it again.
Thanks.

File: ajax/libs/najaxjs/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "basePath": "src",
+      "files": [
+        "+(najax|rlk)*"
+      ]
+    },
+    {
+      "basePath": "bin",
+      "files": [
+        "+(najax|rlk)*"
```
Comment (x09326) [inline ajax/libs/najaxjs/package.json]: @PeterDaveHello 
I'm not sure whether it can be easier like ```"**/*"``` or ```"*.js"```.

Comment (PeterDaveHello) [inline ajax/libs/najaxjs/package.json]: Since it's not `dist` folder, this will be fine.

**Pull request merged**